nuget/Roboto/Uno.Fonts.Roboto.nuspec

```diff
@@ -1,27 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-	
-	<metadata minClientVersion="5.0.0">
-		<id>Uno.Fonts.Roboto</id>
-		<version>1.0.0.0</version>
-		<title>Uno Platform Roboto Font</title>
-		<authors>Uno Platform</authors>
-		<owners>unoplatform</owners>
-    	<license type="expression">Apache-2.0</license>
-		<requireLicenseAcceptance>false</requireLicenseAcceptance>
-		<projectUrl>https://github.com/unoplatform/uno.fonts</projectUrl>
-		<icon>icon.png</icon>
-		<description>Uno Platform Roboto Font</description>
-		<copyright>Copyright (C) 2015-2023 Uno Platform inc. - all rights reserved</copyright>
-		<repository type="git" url="https://github.com/unoplatform/uno.fonts.git" />
-	</metadata>
-
```

Comment:
We'll need to set the same properties in the csproj (https://github.com/unoplatform/Uno.Core/blob/2188706b2550b51cd0d7fe5e735591acc0fe9b95/src/Directory.Build.props#L3-L17)